File: js/easy/squareGrid.js
function grid(count) {
  var cell = '#';
  var s = '';
  for (var i = 0; i < count; i++) {
    if (i % 2 === 0) {
      var row = '';
      for (var j = 0; j < count; j++) {
        if (j % 2 === 0) {
          row += cell;
        } else {
          row += ' ';
        }
      }
      s += row + '\n';
    } else {
      var row = '';
      for (var k = 0; k < count; k++) {
        if (k % 2 === 0) {
          row += ' ';
        } else {
          row += cell;
        }
      }
      s += row + '\n';
    }
  }
  return s;
}
console.log(grid(8));

File: python/easy/square_grid.py
def square_grid(count, cell_char):
  result = ''
  for item in range(1, count + 1):
    if (item % 2 == 0):
      row = ''
      for cell in range(count):
        if (cell % 2 == 0):
          row = row + cell_char
        else:
          row = row + ' '
      result = result + row + '\n'
    else:
      row = ''
      for cell in range(count):
        if (cell % 2 == 0):
          row = row + ' '
        else:
          row = row + cell_char
      result = result + row + '\n'
  return result
print(square_grid(8, '#'))


def repeat_str(s, count):
  result = ''
  for item in range(count):
    result += s
  return result

def paint_square_grid(count=8, symbol='#'):
  result = ''
  odd_str = symbol + ' '
  even_str = ' ' + symbol
  
  for item in range(1,count+1):
    if item % 2 == 0:
      row = repeat_str(even_str, count)
      result += row + '\n'
    else:
      row = repeat_str(odd_str, count)
      result += row + '\n'
  return result
print(paint_square_grid())